datasets/index.md

```diff
@@ -1,14 +1,8 @@
 ---
 layout: default
-title: Data sources
-nav_order: 6
-has_children: false
-redirect_from:
-    - /datasets/covid19
-    - /datasets/international
-    - /datasets/sustainability
-    - /datasets/united_states
-    - /datasets/Disasters
+title: Data Sources
```

Comment:
Please revert; this is to redirect these former pages to the index.

Reply:
Done.